crates/verifier/src/plonk/verify.rs

```diff
@@ -44,7 +44,7 @@ pub(crate) struct PlonkVerifyingKey {
 /// # Returns
 ///
 /// * `Result<bool, PlonkError>` - Returns true if the proof is valid, or an error if verification fails
-pub(crate) fn verify_plonk_raw(
+pub(crate) fn verify_plonk_algebraic(
```

Comment:
add #[must_use] so people don't forget to unwrap

Reply:
must_use is already on by default for `Result`. doing something like this gives me a warning

`crate::PlonkVerifier::verify(...);`